tools/flub/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "@fluid-internal/flub",
```

Comment:
Nit: Is there a reason we are abbreviating the package name? Personally, I think it would be easier to find this package / understand its intended purpose if it were called something like "fluid-bump".

Reply:
The package name could be longer, but the command should be short, because I don't want to have to type a long package name each time I want to run the tool.

"flub" is supposed to be short for "fluid-build".

Maybe @fluid-internal/build-cli? Either way this package probably won't be published, and just used locally.

Reply:
Yeah, wholeheartedly agree with the short command name. My biggest concern is the directory naming (which we generally mostly match in the package name). I think it makes finding things in vscode a lot easier.

Reply:
I renamed the folder to `build-cli` to match the unscoped package name.

Reply:
Thanks!